vulture_os/system/cluster/models.py

```diff
@@ -379,9 +379,8 @@ def get_listeners_enabled(self):
     @property
     def get_forwarders_enabled(self):
         """ Return all tuples (family, proto, ip, port) for each LogForwarders """
-        from applications.logfwd.models import LogOM, LogOMRELP, LogOMHIREDIS, LogOMFWD, LogOMElasticSearch, LogOMMongoDB, LogOMKAFKA
+        from applications.logfwd.models import LogOM, LogOMRELP, LogOMFWD, LogOMElasticSearch, LogOMMongoDB, LogOMKAFKA
         # !!! REQUIRED BY listener_set !
```

Comment:
I guess you can delete this line also, it may have been a limitation in the past.

Reply:
done